Documentation/ColumnsConfig/Type/User/Index.rst

```diff
@@ -161,8 +161,10 @@ implementing a rendering. See :ref:`FormEngine docs
          <t3core:security-xss>`.
 
 
-   The array :php:`$this->data` provides the following data:
```

Comment:
Please restore the original text ...

Reply:
The original text gives no information. Why not make it shorter?

Reply:
The current version is better suited for humans. ;-)